### PR TITLE
feat(HCC-6090): Add che.tender.v1-preview.json

### PR DESCRIPTION
**Pull request by FredrikBlix**

### CODE REVIEW

**Review by NistorNicu — APPROVED** [review]

File: customer-config/checkout-config/che.tender.v1-preview.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    "Tender": {
+      "description": "Tender includes all the forms of payment that are accepted by the retail store\nin settling sales and other transactions.\n",
+      "type": "object",
+      "required": [
+        "id",
+        "currencyId",
+        "businessUnitGroupId"
+      ],
+      "properties": {
+        "businessUnitGroupId": {
```
Comment (FredrikBlix) [inline customer-config/checkout-config/che.tender.v1-preview.json:20]: Should this be businessUnitGroupId or businessUnitId?

Comment (msilfver) [reply customer-config/checkout-config/che.tender.v1-preview.json:20]: The property should not be there at all

File: customer-config/checkout-config/che.tender.v1-preview.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+          "maxLength": 40,
+          "$comment": "PromotionCard"
+        },
+        "description": {
+          "description": "A description for this tender.",
+          "type": "string",
+          "maxLength": 255,
+          "$comment": "A promotion card"
+        },
+        "noAutoFinalization": {
```
Comment (msilfver) [inline customer-config/checkout-config/che.tender.v1-preview.json:60]: Wouldn't it be better to have the name `autoFinalization` and have the default value to true?

File: customer-config/checkout-config/che.tender.v1-preview.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+          "description": "When set to true in the tender data, prevents automatic finalization\nof receipts  upon addition, specifically when both the amount due and change due are 0; if the field is set to false or is null, automatic finalization occurs.",
```
Comment (msilfver) [inline customer-config/checkout-config/che.tender.v1-preview.json:61]: Remove "in the tender data"

File: customer-config/checkout-config/che.tender.v1-preview.json (hunk 2 above)
Comment (msilfver) [reply customer-config/checkout-config/che.tender.v1-preview.json:60]: Maybe have this as an additionalProperty instead. `Pos.autoFinalization`

File: customer-config/checkout-config/che.tender.v1-preview.json (hunk 1 above)
Comment (FredrikBlix) [reply customer-config/checkout-config/che.tender.v1-preview.json:20]: Only the businessUnitGroupId or something more as well?

File: customer-config/checkout-config/che.tender.v1-preview.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.tender.v1-preview.json",
```
Comment (FredrikBlix) [inline customer-config/checkout-config/che.tender.v1-preview.json:3]: Final version should have "che.tender.v1.json"

File: customer-config/checkout-config/che.tender.v1-preview.json (hunk 1 above)
Comment (anwoh) [reply customer-config/checkout-config/che.tender.v1-preview.json:20]: Where do you find the businessUnitGroupId in the tender message then?

**Review by mdovhopo — APPROVED** [review]

**Pull request merged**